### PR TITLE
test: this is a test of the CODEOWNERS behavior for stdlib

### PR DESCRIPTION
**Pull request by nathanielc**
### Done checklist
- [ ] docs/SPEC.md updated
- [ ] Test cases written

### CODE REVIEW

**Review by wolffcm — CHANGES_REQUESTED** [review]
Maybe remove the line that says "do not merge" first

**Comment by nathanielc** [comment]
This should have requested review from the docs team as well. Going to try again as their team didn't have write access to the repo.

**Pull request closed without merging**
